starport/pkg/chaincmd/runner/runner.go

```diff
@@ -150,3 +152,27 @@ func (r Runner) run(ctx context.Context, runOptions runOptions, stepOptions ...s
 
 	return errors.Wrap(err, errb.GetBuffer().String())
 }
+
+func newBuffer() *buffer {
+	return &buffer{
+		Buffer: new(bytes.Buffer),
+	}
+}
+
+type buffer struct {
+	*bytes.Buffer
```

Comment:
```suggestion
	b *bytes.Buffer
```

A good suggestion from the [Uber code style](https://github.com/uber-go/guide/blob/master/style.md#avoid-embedding-types-in-public-structs)

Reply:
uber looks like very opiniated on this. although it does not apply to our case. they recommend avoiding it for public struct in our case, `buffer` is a private one.